packages/react-native/ReactAndroid/src/main/java/com/facebook/react/defaults/DefaultReactHost.kt

```diff
@@ -60,7 +60,7 @@ public object DefaultReactHost {
       jsMainModulePath: String = "index",
       jsBundleAssetPath: String = "index",
       jsBundleFilePath: String? = null,
-      isHermesEnabled: Boolean = true,
+      jsRuntimeFactory: JSRuntimeFactory? = null,
```

Comment:
This is a breaking change @Kudo 

It will have to be in a separate method sadly

Reply:
adding backward compatible methods in https://github.com/facebook/react-native/pull/49366/commits/368d6d29bedc0d0b7745ca267435c396e369aab7